pkg/apis/apps/v1alpha1/helm.go

```diff
@@ -148,6 +148,12 @@ type HelmOptions struct {
 	// +kubebuilder:validation:Type=boolean
 	SkipCRDs *bool `json:"skipCRDs,omitempty"`
 
+	// ReplaceCRDs replace all crds in chart and sub charts before upgrade and install
+	//
+	// +optional
+	// +kubebuilder:validation:Type=boolean
+	ReplaceCRDs *bool `json:"replaceCRDs,omitempty"`
```

Comment:
Please help verify whether this option `ReplaceCRDs` has some conflicts or superpositions with option `Force`.

Reply:
- I think there no conflicts or superpositions with `Force`, `Force` will replace other resources exclude crds 
- `replaceCRDS` may conflict with `skipCRDs`,    when `skipCRDs` is on, `replaceCRDs` should not working

Reply:
Looks good to me.